pubspec.yaml

```diff
@@ -26,7 +26,7 @@ dependencies:
   vibration: ^1.7.6
   visibility_detector: ^0.3.3
   equatable: ^2.0.5
-  hydrated_bloc: ^9.1.1
+  hydrated_bloc: ^9.1.2
```

Comment:
Why is this updated needed? None of the cubits are using the hydrated package

Reply:
VS code popped up a warning about this package specifically so i updated it. I am trusting haha. It is used, but not for this feature. 😀 Its used in ProfilesCubit.
But on that note  I think it could be good for us to take a look at all packages and what is the latest stable version for them.

Reply:
I would create a different pr addressing that issue. Assuming that for this pr we expect changes only for kids-836